components/BasicCard/BasicCard.tsx

```diff
@@ -0,0 +1,122 @@
+import { cnb } from 'cnbuilder';
+import { AnimateInView, type AnimationType } from '@/components/Animate';
+import { NumberCounter } from '@/components/NumberCounter';
+import { Container } from '@/components/Container';
+import { FlexBox } from '@/components/FlexBox';
+import { Heading, type HeadingType, Text } from '../Typography';
+import { accentBorderColors, type AccentBorderColorType, type accentBgColors, type PaddingType } from '@/utilities/datasource';
+import { splitNumberString } from '@/utilities/splitNumberString';
+import { getProcessedImage } from '@/utilities/getProcessedImage';
+import * as styles from './BasicCard.styles';
+
+export type BasicCardProps = React.HTMLAttributes<HTMLDivElement> & {
+  subheading?: string;
+  heading?: string;
+  headingLevel?: HeadingType;
+  isDarkTheme?: boolean;
+  bgColor?: string;
+  barColor?: AccentBorderColorType;
+  body?: React.ReactNode;
+  paddingTop?: PaddingType;
+  imageSrc?: string;
+  imageFocus?: string;
+  imageAspectRatio?: styles.BasicCardImageAspectRatio;
+  cta?: React.ReactNode;
+  isCounter?: boolean;
+  // In number of seconds
+  counterDuration?: number;
+  animation?: AnimationType;
+  delay?: number;
+};
+
+export const BasicCard = ({
+  subheading,
+  heading,
+  headingLevel = 'h3',
+  barColor,
+  bgColor,
+  body,
+  imageSrc = '',
+  imageFocus,
+  imageAspectRatio = '4x3',
+  cta,
+  paddingTop,
+  isDarkTheme,
+  isCounter,
+  counterDuration,
+  animation = 'slideUp',
+  delay,
+  children,
+  className,
+  ...props
+}: BasicCardProps) => {
+  const headingProcessed = isCounter ? splitNumberString(heading) : undefined;
+  const imageSize = imageAspectRatio === '4x3' ? '336x222' : '700x700';
+  const bgCardColor = "bg-" + bgColor;
```

Comment:
@yvonnetangsu I'm getting an error and when I remove the hyphen from the `bg-` the error goes away. This only recently started happening.

Reply:
@jenbreese If you chain a classname this way, they are not going to get rendered because TW needs to see the full class names. Best way to do this is to use the utilities/datasource.ts file to get the classname out of the object. I suggest looking at the `SidebarCard` component to see how this is handled 😄

Reply:
You might see that you're still getting the classes honored here because those `bg-` classnames have appeared in full in another file so they already got compiled, but it is good practice to use the `datasource` file.